lib/src/neu/definitions.ts

```diff
@@ -65,4 +74,19 @@ export interface Body {
   type: Type;
 }
 
+/**
+ * Supported serialization strategies for arrays in query parameters
+ *
+ *    "ampersand": ?id=3&id=4&id=5
+ *    "ampersandEscaped": ?id=3%204%205
+ *    "comma": ?id=3,4,5
+ *    "pipe": ?id=3|4|5
+ */
+export type QueryParamArrayStrategy =
```

Comment:
This looks exactly like the `QueryParamArrayStrategy` defined below, why do we need to declare it again?

https://github.com/airtasker/spot/blob/32237cda11fa7a744d63422f84a1d248700d972e/lib/src/models/types/parameter-serialization-strategies.ts#L9-L14

Reply:
When the generators switch over to the raw parser everything outside of `/neu/ directory will get deprecated, so `neu` needs to be self contained for an easy transition